### DISCUSSION TITLE
How to ask questions about the current open note

### DISCUSSION THREAD

**Discussion by block6791** [brianpetro/obsidian-smart-connections#384] [Q&A]
This seems so simple but I can't figure it out. I would like to ask the Smart Connections Chat questions about the note I have currently open in the middle section of Obsidian. I am using a traditional three pane layout: Folders on the left, note contents in the middle, and the side panel (with Smart Connections chat) on the right. I would like to ask questions on the note that is visible on the middle section, but I cannot figure out how.

I tried to scope the chat question like "Based on the current note,....." and "Based on the note currently open..." but that does not work.

Thanks for helping me out.

**Comment by RealRavens** [comment]
It's a planned feature, if I remember correctly. We just have to keep waiting for this plugin to get a major update. 😔

I'm not by my computer right now but can you supply the obsidian link in the chat and that work?

Else (especially if it's a large note), copy and paste that bad boy in there and say "based on this text" or other equivalent prompts.

**Comment by brianpetro** [comment]
Hi @block6791 

You can specify notes using the [[NOTE LINK]] syntax in the chat. You will have to do this with each message you input into the chat in order to use the same note as context.

Besides that, @RealRavens is correct that the upcoming version includes a toolset for the chat (i.e. actions the AI can trigger instead of responding directly), including "get the currently active note" function, which will enable the type of usage like in your example:

> "Based on the current note,....." and "Based on the note currently open..."

🌴
